src/ontodia/diagram/view.ts

```diff
@@ -36,6 +36,9 @@ export interface DiagramViewOptions {
     linkRouter?: LinkRouter;
     suggestProperties?: PropertySuggestionHandler;
     onIriClick?: (iri: string, element: Element, event: React.MouseEvent<any>) => void;
+    onPointerDown?: (e: PointerEvent) => void;
```

Comment:
Please move this callbacks to the `WorkspaceProps` and subscribe to corresponding events in the `Workspace`.